### DISCUSSION TITLE
Feature request - set pet as indoor pet

### DISCUSSION THREAD

**Discussion by BinnenBeest** [benleb/surepy#128] [Ideas]
If it would be possible to toggle the indoor pet status of my cats that should be great. 

The standard app doesn’t allow me to limited a cat to go outside for some hours. 
![D27A8EBA-84A4-49ED-93A1-3E15F50E06A0](https://user-images.githubusercontent.com/61503030/168981857-abe67e29-54af-46fa-8da6-5c2f4a982f27.jpeg)

It’s all or nothing so I have to adapt this multiple times a day myself. 

With home assistant I would like to script this ex. free to go outside from 8AM till 10AM for one particular pet. 

Thanks in advanced for considering this feature.

**Comment by jspiekstra** [comment]
I have added this functionality. I will request allowance for push request to @benleb so it hopefully can be merged into the next release.
